lib/hammer_cli_foreman/exception_handler.rb

```diff
@@ -83,16 +83,43 @@ def handle_foreman_error(e)
       log_full_error e
     end
 
-    def handle_apipie_docloading_error(e)
-      rake_command = "foreman-rake apipie:cache"
-      message = _("Could not load the API description from the server") + ":"
-      message += "\n#{e.original_error.message}" if e.respond_to?(:original_error)
-      message += "\n  - " +
-                 _("is the server down?") + "\n  - " +
-                 _("was '%s' run on the server when using apipie cache? (typical production settings)") % rake_command
-      print_error message
-      log_full_error e
-      HammerCLI::EX_CONFIG
+    def ssl_cert_instructions
+      host_url = HammerCLI::Settings.get(:_params, :host) || HammerCLI::Settings.get(:foreman, :host)
+      uri = URI.parse(host_url)
+      if uri.host.nil?
+        host = '<FOREMAN_HOST>'
+        cert_name = "#{host}.crt"
+      else
+        host = uri.host
+        host += ":#{uri.port}" if uri.port
+        cert_name = "#{uri.host}.crt"
+      end
+
+      cmd = "hammer --fetch-ca-cert #{host}"
+
+      rh_install_cmd = "install #{cert_name} /etc/pki/ca-trust/source/anchors/"
+      rh_update_cmd = "update-ca-trust"
+      deb_install_cmd = "install #{cert_name} /usr/local/share/ca-certificates/"
+      deb_update_cmd = "update-ca-certificates"
+
+      _("Make sure you configured the correct URL and have the server's CA certificate installed on your system.") + "\n\n" +
+      _("You can use hammer to fetch the CA certificate from the server. Be aware that hammer cannot verify whether the certificate is correct and you should verify its authenticity after downloading it.") +
+      "\n\n" +
+      _("Download the certificate to a secure location and save it as a CRT file as follows:") +
+      "\n\n  $ #{cmd} > #{cert_name}\n\n" +
```

Comment:
I got in the output:
```
Download the certificate to a secure location and save it as a CRT file as follows:

  $ hammer --fetch-ca-cert bkr-hv01-guest17.dsal.lab.eng.bos.redhat.com:443 > bkr-hv01-guest17.dsal.lab.eng.bos.redhat.com.crt
```

When proceeded with that I got:

```
$ hammer --fetch-ca-cert bkr-hv01-guest17.dsal.lab.eng.bos.redhat.com:443 > bkr-hv01-guest17.dsal.lab.eng.bos.redhat.com.crt
Couldn't parse URI 'bkr-hv01-guest17.dsal.lab.eng.bos.redhat.com:443'.
Perhaps you meant to connect to 'https://bkr-hv01-guest17.dsal.lab.eng.bos.redhat.com:443'?
```

We should either  mention the protocol in the output or fix --fetch-ca-cert to auto add it when missing.